### PR TITLE
Change: Improve feed sync documentation for source build

### PR DESCRIPTION
**Pull request by bjoernricks**
**What**:

Improve feed sync documentation for source build

**Why**:

Make the feed sync more prominent in the source build documentation and
explain the sync process in more detail to avoid issues users are having
at the moment.

**How**:

**Checklist**:

- [ ] Tests
- [x] CHANGELOG.md entry
- [ ] Documentation

### CODE REVIEW

File: src/common/source-build/feed-sync-other.rst [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+------------------------------------
+
+:term:`SCAP` data contains :term:`CPE` and :term:`CVE` information.
+
+.. note::
+
+  The CERT feed synchronization depends on the data provided by the SCAP
+  feed and should be called after synchronizing the latter. The GVMD_DATA
+  contains scan configs which depend on the VTs. Only if the VTs are synced and
+  loaded scan configs are available and scans can be started.
```
Comment (k-schlosser) [inline src/common/source-build/feed-sync-other.rst:11]:
```suggestion
  loaded scan configs are available, scans can be started.
```

File: src/common/source-build/feed-sync-other.rst [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+.. code-block::
+  :caption: Syncing SCAP data processed by gvmd, this will take a while
+
+  sudo -u gvm greenbone-feed-sync --type SCAP
+
+CERT data contains vulnerability information from the German `DFN-CERT <https://www.dfn-cert.de/>`_
+and `CERT-Bund <https://cert-bund.de/>`_ agencies.
```
Comment (k-schlosser) [inline src/common/source-build/feed-sync-other.rst:19]:
```suggestion
and `CERT-Bund <https://www.bsi.bund.de/EN/Themen/Unternehmen-und-Organisationen/Cyber-Sicherheitslage/Reaktion/CERT-Bund/cert-bund_node.html>`_ agencies.
```

File: src/common/source-build/feed-sync-other.rst [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+.. code-block::
+  :caption: Syncing CERT data processed by gvmd
+
+  sudo -u gvm greenbone-feed-sync --type CERT
+
+gvmd data (or also called data-objects) are scan configurations, compliance policies, port lists
```
Comment (k-schlosser) [inline src/common/source-build/feed-sync-other.rst:26]:
```suggestion
gvmd data (or also called "data objects") are scan configurations, compliance policies, port lists
```

File: src/common/source-build/feed-sync.rst [hunk 4]
```diff
@@ -0,0 +1,19 @@
+Performing a Feed Synchronization
+=================================
+
+For the actual vulnerability scanning, :term:`Vulnerability Test scripts<VT>`,
+security information like :term:`CVEs <CVE>`, port lists and scan configurations
+are required. All this data is provided by the :term:`Greenbone Community Feed`
+and should to be download initially before starting the services.
```
Comment (k-schlosser) [inline src/common/source-build/feed-sync.rst:7]:
```suggestion
and should be downloaded initially before starting the services.
```

File: src/common/source-build/feed-sync.rst [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+
+A synchronization always consists of two parts:
+
+1. Downloading the changes via :command:`rsync`
+2. The changes get loaded into memory and a database by a daemon
```
Comment (k-schlosser) [inline src/common/source-build/feed-sync.rst:12]:
```suggestion
2. Loading the changes into memory and a database by a daemon
```

**Review by k-schlosser — CHANGES_REQUESTED** [review]

File: src/common/source-build/feed-loading.rst [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+  take a few minutes, please wait...
+
+The loading of the VT data is finished if the log message can be found:
+
+.. code-block:: none
+  :caption: ospd-openvas VTs loading finished log message
+
+  Finished loading VTs. The VT cache has been updated from version X to Y.
+
+After the scanner is aware of the VT data, it will be requested by gvmd. This
```
[the remark this answers is not in the document]
Comment (bjoernricks) [reply src/common/source-build/feed-loading.rst:21]: the VT data is meant here. After the scanner has loaded the VT information into the redis database, gvmd can request this information from ospd-openvas. Before gvmd will get a message from ospd-openvas that the VT information is not available yet.

**Review by k-schlosser — APPROVED** [review]

**Pull request merged**